src/lexer.coffee

```diff
@@ -473,7 +476,11 @@ exports.Lexer = class Lexer
         @error message, origin[2] if message
       return value.length if skipToken
 
-    if value is '{' and prev?[0] is 'EXPORT'
+    if value is '{' and @seenImport
+      @importSpecifierList = yes
+    else if @importSpecifierList and value is '}'
```

Comment:
@vendethiel the “destructuring” in ES2015 `import` statements is just a [similar-looking syntax](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&code=import%20%7B%20a%3A%20b%20%7D%20from%20'lib'):
```js
import { a: b } from 'lib'
```
```
repl: ES2015 named imports do not destructure. Use another statement for destructuring after the import. (1:12)
> 1 | import { a: b } from 'lib'
    |             ^
```